docs/terraform.md

```diff
@@ -35,11 +35,21 @@
 
 | Name | Description |
 |------|-------------|
+<<<<<<< Updated upstream
```

Comment:
@bnippercg This is the merge conflict that got checked in. Please address.

Reply:
Not sure how I missed that, I'll be sure to correct that before the next update.